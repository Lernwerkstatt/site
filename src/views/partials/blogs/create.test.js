const form = require("./create.hbs");

describe("create.hbs", () => {
  it("should match the snapshot", () => {
    expect(form({})).toMatchSnapshot();
  });
});
