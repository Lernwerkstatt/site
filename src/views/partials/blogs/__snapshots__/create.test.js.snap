// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create.hbs should match the snapshot 1`] = `
"<form action=\\"/blogs/create\\" method=\\"post\\" autocomplete=\\"off\\" enctype=\\"application/json\\">
  <div class=\\"form-group col-md-6\\">
    <label>Titel:</label>
    <input type=\\"text\\" class=\\"form-control\\" name=\\"title\\" required>
  </div>
  <div class=\\"form-group col-md-6\\">
    <label>Autor:</label>
    <input type=\\"text\\" class=\\"form-control\\" name=\\"author\\" required>
  </div>
  <div class=\\"form-group col-md-6\\">
    <label>Inhalt:</label>
    <textarea name=\\"content\\" class=\\"form-control\\" rows=\\"10\\" required></textarea>
  </div>
  <div class=\\"form-group col-md-6\\">
    <label>Link f√ºr das Titelbild:</label>
    <input type=\\"text\\" class=\\"form-control\\" name=\\"imagelink\\">
  </div>
  <div class=\\"col-md-6\\">
    <input type=\\"submit\\" value=\\"Submit\\">
    <input type=\\"reset\\">
  </div>
</form>
"
`;
